cockatrice/src/settingscache.h

```diff
@@ -96,6 +96,8 @@ class SettingsCache : public QObject {
     bool getChatMention()  const { return chatMention; }
     bool getZoneViewSortByName() const { return zoneViewSortByName; }
     bool getZoneViewSortByType() const { return zoneViewSortByType; }
+    bool getZoneViewPileView() const { return zoneViewPileView; }
+    bool getZoneViewShuffle() const { return zoneViewShuffle; }
```

Comment:
I think some doc comments on these methods would be good - the name doesn't really tell me what it's used for